stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/suggestion/list/SuggestionListViewStyle.kt

```diff
@@ -0,0 +1,177 @@
+package io.getstream.chat.android.ui.suggestion.list
+
+import android.content.Context
+import android.graphics.Typeface
+import android.graphics.drawable.Drawable
+import android.util.AttributeSet
+import androidx.annotation.ColorInt
+import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.TransformStyle
+import io.getstream.chat.android.ui.common.extensions.internal.getColorCompat
+import io.getstream.chat.android.ui.common.extensions.internal.getDimension
+import io.getstream.chat.android.ui.common.extensions.internal.getDrawableCompat
+import io.getstream.chat.android.ui.common.extensions.internal.use
+import io.getstream.chat.android.ui.common.style.TextStyle
+
+/**
+ ** Style for SuggestionListView
```

Comment:
```suggestion
 ** Style for [SuggestionListView]
```
but probably you'll have to import this class